packages/main/test/specs/CheckBox.spec.js

```diff
@@ -87,4 +87,14 @@ describe("CheckBox general interaction", () => {
 			assert.strictEqual(await defaultPreventedCb.getProperty("indeterminate"), state.indeterminate, "The checkbox indeterminate is not changed");
 		}
 	});
+
+	it("tests form submission when third checkbox is checked and button is clicked", async () => {
```

Comment:
This test is OK, but it would be nice to test the prevention of submition, if the checkbox is not checked.

Reply:
Done.